tests/rule_based_profiler/domain_builder/test_domain_builder.py

```diff
@@ -14,8 +14,12 @@
     ParameterContainer,
     build_parameter_container_for_variables,
 )
+from tests.integration.profiling.rule_based_profilers.conftest import (
+    alice_columnar_table_single_batch,
+    alice_columnar_table_single_batch_context,
+)
```

Comment:
Ah this is necessary because the fixtures are for the integration tests? Could we add a small comment noting why we're doing importing from `conftest` just for some clarity?

Reply:
I think I have convinced myself that it is correct to keep the integration tests in a separate directory like this (we need to test integration of various parts of the codebase). I'm just wondering if we should duplicate the fixtures or create new ones so we aren't importing between integration and unit tests. I think creating silos between test directories as much as possible is preferred due to the shared state issues we want to avoid (@cdkini please correct me if this understanding is wrong).